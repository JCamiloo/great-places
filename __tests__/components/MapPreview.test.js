import React from "react";
import MapPreview from "../../components/MapPreview";
import renderer from "react-test-renderer";

describe("MapPreview component tests", () => {
  test("MapPreview component renders correctly", () => {
    const tree = renderer.create(<MapPreview />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
