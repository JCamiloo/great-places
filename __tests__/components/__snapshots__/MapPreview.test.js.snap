// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapPreview component tests MapPreview component renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "opacity": 1,
    }
  }
/>
`;
